pandas/tests/frame/test_operators.py

```diff
@@ -829,6 +829,28 @@ def test_inplace_ops_identity2(self, op):
         expected = id(df)
         assert id(df) == expected
 
+    def test_inplace_drop_and_add(self):
+        # GH 30484
+        # Get expected df
+        expected = pd.DataFrame({})
+        expected["x1"] = [1, 2, 3, 4, 5]
+        expected["x2"] = [0, 0, 0, 1, 1]
```

Comment:
test is very hard to read, also you can just directly construct the dataframes with dict of lists

Reply:
More readable test implemented in the new commit